xsd/netex_framework/netex_reusableComponents/netex_seatingPlan_support.xsd

```diff
@@ -206,23 +210,32 @@
 			</xsd:restriction>
 		</xsd:simpleContent>
 	</xsd:complexType>
+	<!-- ===== LOCATABLE SPOT RANGE ======================================================== -->
 	<xsd:group name="LocatableSpotRangeGroup">
 		<xsd:annotation>
 			<xsd:documentation>A Range of LOCATABLE SPOTs. </xsd:documentation>
 		</xsd:annotation>
 		<xsd:sequence>
-			<xsd:element name="StartSpot" type="locatableSpotRefs_RelStructure" minOccurs="0">
+			<xsd:element name="StartSpot" type="LocatableSpotRangeRefStructure" minOccurs="0">
 				<xsd:annotation>
 					<xsd:documentation>Start of range of LOCATABLE SPOTs.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
-			<xsd:element name="EndSpot" type="locatableSpotRefs_RelStructure" minOccurs="0" maxOccurs="unbounded">
+			<xsd:element name="EndSpot" type="LocatableSpotRangeRefStructure" minOccurs="0" maxOccurs="unbounded">
 				<xsd:annotation>
 					<xsd:documentation>End of range of LOCATABLE SPOTs.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
 		</xsd:sequence>
 	</xsd:group>
+	<xsd:complexType name="LocatableSpotRangeRefStructure">
```

Comment:
```suggestion
	<xsd:complexType name="LocatableSpotRefStructure">
```

Reply:
I suggest to remove "Range" because it references a single spot, not a  range.

Reply:
@trurlurl are you sure that this changes would validate?

Reply:
> @trurlurl are you sure that this changes would validate?

The structure is freshly introduced, i.e., hasn't existed prior to this PR, and I haven't found it being used in the examples.